web/src/routes/timetable/[type]/[[id]]/+page.svelte

```diff
@@ -0,0 +1,157 @@
+<script lang="ts">
+  import { goto } from '$app/navigation';
+  import * as Tabs from '$lib/elements/ui/tabs/index';
+  import ComboBox from '$lib/elements/ui/combo-box/combo-box.svelte';
+  import {
+    getClassTimetableLessons,
+    getRoomTimetableLessons,
+    getStudentTimetableLessons,
+    getTeacherTimetableLessons,
+    resourceDisplayName,
+  } from '$lib/components/timetable/timetable.js';
+  import {
+    getRoomsItems,
+    getStudentGroupsItems,
+    getStudentsItems,
+    getTeachersItems,
+  } from '$lib/utils/combobox-items.js';
+  import type { ComboBoxItem } from '$lib/elements/ui/combo-box/combo-box.js';
+  import Timetable from '$lib/components/timetable/timetable.svelte';
+  import {
+    getRoomLessons,
+    getStudentGroupLessons,
+    getStudentLessons,
+    getTeacherLessons,
+  } from '$lib/sdk/fetch-client.js';
+
+  export let data;
+  let items: ComboBoxItem[] = [];
+  let selectedId: string = '';
+
+  $: id = data.id;
+  $: selectedId = items.map(({ value }) => value).includes(selectedId) ? selectedId : items.at(0)?.value || '';
+
+  const onTimetableSearch = (value: string) => {
+    items = items.filter((item) => item.label.toLowerCase().includes(value.toLowerCase()));
```

Comment:
Wouldn't this result in the search only working correctly one time and removing the search parameter wouldn't display all items again, as `items` gets overwritten and thus loosing the information about all items that were removed.

Reply:
Oh I think you are correct. This does sound weird

Reply:
Fixed, thanks!